src/mxml/dom/Bracket.h

```diff
@@ -0,0 +1,59 @@
+//
+//  Bracket.h
+//  pianomarvel
+//
+//  Created by nhatlee on 2/6/18.
+//  Copyright © 2018 nhatlee. All rights reserved.
+//
```

Comment:
Can you please replace with the standard header found in other files. You can leave your name in, of course.